cfdisk: Compile Fails

Normal:

```
./autogen.sh
./configure
make
```

results in:

```
  CC     disk-utils/cfdisk-cfdisk.o
disk-utils/cfdisk.c: In function ‘ui_init’:
disk-utils/cfdisk.c:700:2: error: too few arguments to function ‘colors_init’
./include/colors.h:55:12: note: declared here
```

--- a/disk-utils/cfdisk.c
+++ b/disk-utils/cfdisk.c
@@ -697,7 +697,7 @@ static int ui_init(struct cfdisk *cf __attribute__((__unused__)))
 			init_pair(i, color_pairs[i][0], color_pairs[i][1]);
 	}
 #else
-	colors_init(UL_COLORMODE_NEVER);
+	colors_init(UL_COLORMODE_NEVER, "cfdisk");
 #endif
 
 	cbreak();

